xde/src/mac.rs

```diff
@@ -176,6 +178,28 @@ impl MacClientHandle {
         unsafe { mac_rx_set(self.mch, rx_fn, arg) };
     }
 
+    pub fn add_unicast(
+        self: &Arc<Self>,
+        mac: EtherAddr,
+    ) -> Result<MacUnicastHandle, c_int> {
+        let mut diag = mac_diag::MAC_DIAG_NONE;
+        let mut ether = mac.to_bytes();
+        let mut muh = 0 as *mut mac_unicast_handle;
```

Comment:
nit: `ptr::null_mut()` instead of casting from 0

Reply:
Done